setup.py

```diff
@@ -32,6 +32,7 @@
         'numpy >= 1.10',
         'matplotlib >= 1.5',
         'urllib3 >= 1.21.1',
+        'antlr4-python3-runtime>=4.7',
```

Comment:
How big of a dependency is this, in terms of ease of getting? Does it pip install fine?

Reply:
Compressed size: 107KB
It installed fine on my machine, and it's pure python with no dependencies.

The tricky part is that there's also `antlr4-python2-runtime` which we would need to switch between.